setup.py

```diff
@@ -16,13 +16,13 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
-    python_requires='>=3.7',
+    python_requires=">=3.7",
```

Comment:
Please replace this file with [this one](https://github.com/fastai/nbdev/blob/master/setup.py)

Reply:
replaced the file